package.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "react-mobile-picker",
-  "version": "0.1.12",
+  "name": "@maxiru/react-mobile-picker",
```

Comment:
please keep the basic info such as `name`, `version`, `repositiry` in `package.json`. Of course you can modify the `contributors` field to add you name.